pipeline/data_page/get_all_data_availability.py

```diff
@@ -66,29 +67,37 @@ def preprocess_omics_dataframe(df, dataset_id):
     4. Set ModelID as index
     5. Drop columns with all NaN values
     """
```

Comment:
@snwessel The Long Reads files now have this IsDefaulyEntryForModel file. This makes me wonder if the datasets need to go through this `preprocess_omics_dataframe` step. This is something you should probably double check with Phil before deploying these changes to prod.

Reply:
Oh right, that's a good point, thank you! In that case, it might be good to get @naquib314 's review on this too when he's back on Tuesday - he's probably the most familiar with how these new omics indices are handled here